### PR TITLE
Update guide for installing the latest Release Candidate (5.4)

### PR DESCRIPTION
**Pull request by zoglo**
### Description

* Updates the installation guide for test versions _(were outdated and for 4.9)_

___Further notice:___
_I've already updated the assets folder to `public/assets` to match the structure in `^5.4`_

### CODE REVIEW

File: docs/manual/guides/install-test-versions.de.md [hunk 1]
```diff
@@ -175,7 +172,7 @@ Version wird, sieht so aus:
         }
     },
     "extra": {
-        "contao-component-dir": "assets"
+        "contao-component-dir": "public/assets"
```
Comment (fritzmg) [inline docs/manual/guides/install-test-versions.de.md:175]: Hm, the `contao/managed-edition` still has only `assets`:

https://github.com/contao/managed-edition/blob/3039b6e040c54081033ee330b203398773d158e9/composer.json#L30-L32

Is this a bug in the `contao/managed-edition` repository?

Comment (zoglo) [reply docs/manual/guides/install-test-versions.de.md:175]: Might be, the branch for 5.4 didn't exist yesterday (https://github.com/contao/managed-edition/commit/3039b6e040c54081033ee330b203398773d158e9)

Comment (fritzmg) [reply docs/manual/guides/install-test-versions.de.md:175]: Also the `5.x` branch only has `assets`. Can you make a PR for `contao/managed-edition` first where we can confirm that this is actually correct?

Comment (fritzmg) [reply docs/manual/guides/install-test-versions.de.md:175]: Also wouldn't this break for everyone updating from 4.13 or 5.3 to 5.4? 🤔

Comment (zoglo) [reply docs/manual/guides/install-test-versions.de.md:175]: Well, the guide is for installing the latest test version aka RC.

I did have some lovely issues updating a 5.3 demo to 5.4, would need to test it one more time.
assets doesn't get moved to public, however the setup does handle it afaik

Comment (fritzmg) [reply docs/manual/guides/install-test-versions.de.md:175]: > Well, the guide is for installing the latest test version aka RC.

Yes, but both the `5.4` branch and `5.x` branch of the `contao/managed-edition` do not contain this change.

Comment (zoglo) [reply docs/manual/guides/install-test-versions.de.md:175]: https://github.com/contao/managed-edition/pull/63 <-- leaving that for documentation

Comment (zoglo) [reply docs/manual/guides/install-test-versions.de.md:175]: > I did have some lovely issues updating a 5.3 demo to 5.4, would need to test it one more time.

 Guess it was https://github.com/contao/contao/issues/7354 :)

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @zoglo :)

**Pull request merged**